sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -218,6 +218,30 @@ struct ImageDeleter {
   }
 };
 
+static bool is_compiler_available(const context &C,
```

Comment:
Let's update the name of the function to something like `is_device_binary_type_supported`.